src/app/static/src/app/store/stepper/getters.ts

```diff
@@ -20,7 +20,7 @@ export const getters: StepperGetters & GetterTree<StepperState, RootState> = {
     complete: (state: StepperState, getters: any, rootState: RootState, rootGetters: any) => {
         return {
             1: rootGetters['baseline/complete'] && rootGetters['surveyAndProgram/complete'],
-            2: rootGetters['baseline/complete'] && rootGetters['metadata/complete'] && rootGetters['surveyAndProgram/complete'],
```

Comment:
So this change will mean they can advance straight to step 3 from step 1 right? No need to go to 2nd step and wait for data to be fetched?

Reply:
yep, but that shouldnt break anything right, was just the behaviour before